crates/db_schema/src/impls/site.rs

```diff
@@ -18,11 +18,6 @@ impl Crud for Site {
   type UpdateForm = SiteUpdateForm;
   type IdType = SiteId;
 
-  /// Use SiteView::read_local, or Site::read_from_apub_id instead
-  async fn read(_pool: &mut DbPool<'_>, _site_id: SiteId) -> Result<Self, Error> {
-    unimplemented!()
-  }
```

Comment:
I would leave this implementation as is, to avoid people calling it unintentionally.

Reply:
Fixed